app/locales/es.json

```diff
@@ -137,6 +137,8 @@
     "location_match_description": "Comprueba si has tenido contacto con alguien infectado",
     "logging_active": "Localización activada",
     "logging_inactive": "Localización desactivada",
+    "share_loc_active": "Estás compartiendo tu ubicación con el Ministerio de Salud",
```

Comment:
The resources are order alphabetically, please follow the convention and order your resource alphabetically